specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2021-04-01/ThreatIntelligence.json

```diff
@@ -0,0 +1,1183 @@
+        },
+        "value": {
+          "description": "Array of threat intelligence information objects.",
+          "items": {
+            "$ref": "#/definitions/ThreatIntelligenceInformation"
+          },
+          "type": "array"
+        }
+      },
+      "type": "object",
+      "required": [
+        "value"
+      ]
+    },
+    "ThreatIntelligenceIndicatorModel": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/ThreatIntelligenceInformation"
+        }
+      ],
+      "description": "Threat intelligence indicator entity.",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ThreatIntelligenceIndicatorProperties",
+          "description": "Threat Intelligence Entity properties",
+          "x-ms-client-flatten": true
+        }
+      },
+      "type": "object",
+      "x-ms-discriminator-value": "indicator"
+    },
+    "ThreatIntelligenceIndicatorModelForRequestBody": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/ThreatIntelligenceResourceKind"
+        }
+      ],
+      "description": "Threat intelligence indicator entity used in request body.",
+      "properties": {
+        "etag": {
+          "description": "Etag of the azure resource",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/ThreatIntelligenceIndicatorProperties",
+          "description": "Threat Intelligence Entity properties",
+          "x-ms-client-flatten": true
+        }
+      },
+      "type": "object",
+      "x-ms-discriminator-value": "indicator"
+    },
+    "ThreatIntelligenceResourceKind": {
+      "description": "Describes an entity with kind.",
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/ThreatIntelligenceResourceInnerKind",
+          "description": "The kind of the entity."
+        }
+      },
+      "required": [
+        "kind"
+      ],
+      "type": "object"
+    },
+    "ThreatIntelligenceResourceInnerKind": {
+      "description": "The kind of the threat intelligence entity",
+      "enum": [
+        "indicator"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "ThreatIntelligenceResourceInnerKind",
+        "values": [
+          {
+            "description": "Entity represents threat intelligence indicator in the system.",
+            "value": "indicator"
+          }
+        ]
+      }
+    },
+    "ThreatIntelligenceIndicatorProperties": {
+      "allOf": [
+        {
+          "$ref": "../../../common/1.0/types.json#/definitions/EntityCommonProperties"
+        }
+      ],
+      "description": "Describes threat intelligence entity properties",
+      "properties": {
+        "threatIntelligenceTags": {
+          "description": "List of tags",
+          "items": {
+            "description": "tag",
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "lastUpdatedTimeUtc": {
```

Comment:
>lastUpdatedTimeUtc [](http://example.com/codeflow?start=9&length=18)

This should be implemented as part of systemData.

Wiki link: [https://armwiki.azurewebsites.net/api_contracts/ResourceSystemData.html?q=systemdata](https://armwiki.azurewebsites.net/api_contracts/ResourceSystemData.html?q=systemdata)

Reply:
Previously reviewed.